README.md

```diff
@@ -1,5 +1,95 @@
 # ActiveRecord CRUD Intro
 
+## Objectives
 
-1. Explain Create, Read, Update, Delete 
-2. Add codesnippets to the explanations as examples
\ No newline at end of file
+
+1. Understand CRUD - Create, Read, Update, and Delete.
+2. Get familiar with how to use ActiveRecord methods for Crud.
+
+## What is CRUD?
+
+CRUD is an acronym for the four verbs we use to operate on data: **C**reate, **R**ead, **U**pdate and **D**elete. Active Record automatically creates methods to allow an application to read and manipulate data stored within its tables.
+
+### Create
+
+To create new objects, we can either use the `new` or `create` method. 
+
+The `new` method will create a new object without any attributes.
+
+```ruby
+cat = Cat.new
+```
+To add attributes to our cat instance we need to write:
+
+```ruby
+cat = Cat.new
+cat.name = "Maru"
+cat.age = 3
+```
+The new method instantiates an object without saving it in the database. To persist the data we need to save it.
+
+```ruby
+cat = Cat.new
+cat.name = "Maru"
+cat.age = 3
+cat.save
+```
+
+The `create` Method will create an object with its attributes and save it to the database.
```

Comment:
Add a line: "We pass in the attributes using hash syntax. The key is the name of the column we're updating and the value is what we're changing it to."